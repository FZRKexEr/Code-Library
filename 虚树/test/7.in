10
1 2 6432
2 3 9694
3 4 9714
4 5 8890
4 6 9170
1 7 6361
1 8 1627
2 9 9824
7 10 9947
5
3 4 7 2
5 7 2 10 6 3
5 3 7 4 8 10
3 3 2 10
2 6 8
