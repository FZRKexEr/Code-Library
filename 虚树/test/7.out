12793
12793
14420
12793
8059
